Sources/Commands/SwiftTool.swift

```diff
@@ -758,6 +758,9 @@ public class SwiftTool {
             // binary targets.
             let dataDir = try self.getActiveWorkspace().location.workingDirectory
             let builtToolsDir = dataDir.appending(components: try self._hostToolchain.get().triple.platformBuildPathComponent(), buildEnvironment.configuration.dirname)
+            
+            // Use the directory containing the compiler as an additional search directory.
+            let toolSearchDirs = [try self.getToolchain().swiftCompilerPath.parentDirectory]
```

Comment:
should this made into a func so that can be reused in SwiftPackageTool or other tools that may need this?